Resources/Prototypes/Entities/Clothing/Masks/masks.yml

```
@@ -27,6 +27,21 @@
     sprite: Clothing/Mask/gassecurity.rsi
   - type: Clothing
     sprite: Clothing/Mask/gassecurity.rsi
+  - type: ExplodeOnTrigger
+  - type: Explosive
+    explosionType: Default
+    maxIntensity: 10
+    intensitySlope: 3
+    totalIntensity: 500 # about a ~4 tile radius
```

не совсем понимаю, что за взрывная маска, но 500 - это очень много, и коммент неверен -
радиус взрыва в ~4 тайла это
```suggestion
    totalIntensity: 120 # about a ~4 tile radius
```
maxIntensity, как я понял, отвечает за кол-во урона на каждый тайл, а Slope - насколько быстро он падает по мере отдаления от эпицентра взрыва. Здесь они равны обычной гранате.
для сравнения, totalIntensity паучьего заряда (ниндзя) равняется 360, а гранате "критическая масса" - 20000 (радиус в ~15 тайлов).

---

Это просто тестовая штука для демонстрации новичкам, надо удалить изменение